src/overlays/actors/ovl_En_Ossan/z_en_ossan.h

```diff
@@ -6,7 +6,7 @@
 #include "objects/object_ani/object_ani.h"
 #include "objects/object_fsn/object_fsn.h"
 
-#define ENOSSAN_LIMB_MAX MAX(FSN_LIMB_MAX + 1, ANI_LIMB_MAX)
+#define ENOSSAN_LIMB_MAX MAX((u32)FSN_LIMB_MAX + 1, (u32)ANI_LIMB_MAX)
```

Comment:
Why the + 1?

Reply:
This is because of a bug in `object_fsn`which is noted in `object_fsn.xml`. tldr, the model only has 17 limbs, but the skeleton declares it as having 18.
Though I should add a comment referencing it.